### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by jasonrees1**

### CODE REVIEW

File: jrportfolio.css [hunk 1]
```diff
@@ -301,55 +305,62 @@ html {
...
-	width: 85%;
-	font-size: 1.6rem;
-	margin-top: 17px;
-}
-
-.personalinfo p {
-	line-height: 36px;
-}
-
-}
```
Comment (ashleycoles) [inline jrportfolio.css:355]: Looks like your 768px media query is missing its closing curly bracket.

File: jrportfolio.css [hunk 2]
```diff
@@ -301,55 +305,62 @@ html {
...
 	}
 /* MQ's for about me and contact me sections */
-.myinfocontainer {
-	width: 100%;
-	margin: 0 auto;
-	overflow: auto;
-}
-
-.buttoncontainer {
+	.buttoncontainer {
```
Comment (ashleycoles) [inline jrportfolio.css:362]: Check this and make sure it's indented correctly.

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: jrportfolio.css (hunk 1 above)
Comment (jasonrees1) [reply jrportfolio.css:355]: Done - I think

File: jrportfolio.css (hunk 2 above)
Comment (jasonrees1) [reply jrportfolio.css:362]: Done

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**